tests/test_enumeration.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import re
+import pytest
+import xml.etree.ElementTree as et
+import lsst.ts.xml as ts_xml
+
+
+@pytest.mark.parametrize("xmlfile,csc,topic", ts_xml.get_xmlfile_csc_topic())
+def test_enumeration(xmlfile, csc, topic):
+    """Test that enumerations have the correct format.
+
+    Parameters
+    ----------
+    xmlfile : `pathlib.Path`
+        Full filepath to the Events XML file for the CSC.
+    csc : `testutils.subsystems`
+        Name of the CSC
+    """
+    # Test the topic <EFDB_Name> field.
+    with open(str(xmlfile), "r", encoding="utf-8") as f:
+        tree = et.parse(f)
+    root = tree.getroot()
+
+    reg_exp = re.compile("^[A-Z][a-zA-Z0-9_]\\s?=?\\s?(\\d)?")
```

Comment:
Please consider the following because it is pickier about the values. It works for all the cases I could find. It insists that hex values start with 0x or 0X, which I think is reasonable. The leading "r" means you don't have to double the backslashes.
```
r"^[A-Z][a-zA-Z0-9_]*(\s*=\s*((0[xX][0-9a-fA-F]+)|(\d+)))?$"
```